From 5de575f69729f48fc8f3a1b1c0c7337c9b678e1a Mon Sep 17 00:00:00 2001
From: duxy <568988005@qq.com>
Date: Thu, 21 May 2020 12:37:07 +0800
Subject: [PATCH] [fix] no need to check the temp because of the temp only can
 record 0~125

---
 drivers/thermal/nxp_tmu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/nxp_tmu.c b/drivers/thermal/nxp_tmu.c
index a4b8f66..cdd3718 100755
--- a/drivers/thermal/nxp_tmu.c
+++ b/drivers/thermal/nxp_tmu.c
@@ -146,7 +146,7 @@ int nxp_tmu_get_temp(struct udevice *dev, int *temp)
 		printf("invalid data\n");
 		return ret;
 	}
-
+	/* because of the tmp only can record 0~125,  when temp under 0 it is will > alert
 	while (cpu_tmp >= pdata->alert) {
 		printf("CPU Temperature (%dC) has beyond alert (%dC), close to critical (%dC)",
 		       cpu_tmp, pdata->alert, pdata->critical);
@@ -158,7 +158,7 @@ int nxp_tmu_get_temp(struct udevice *dev, int *temp)
 			return ret;
 		}
 	}
-
+	*/ 
 	*temp = cpu_tmp / 1000;
 
 	return 0;
-- 
2.10.1

